fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/F-PLN/MfdFmsFplnDirectTo.tsx

```diff
@@ -164,7 +164,7 @@ export class MfdFmsFplnDirectTo extends FmsPage<MfdFmsFplnDirectToProps> {
                     containerStyle="width: 175px;"
                     alignLabels="flex-start"
                     onModified={(i, text) => {
-                      if (i) {
+                      if (i != null) {
```

Comment:
```suggestion
                      if (i !== null && i !== undefined) {
```
Thanks for the fix, well spotted! We should avoid juggling checks, as they could be perceived as unintentional. I'd rather check for null and undefined explicitly

Reply:
Saw it in the live stream with capta320 and riiiva, in which you were also present. I asked myself exactly whether I should do an explicit null check. Thx for the feedback, even if it is a small change.

Reply:
Sure, thanks for the fix. We will include it in version 0.12.2, which we're going to release soon.